tests/test_client.py

```diff
@@ -37,13 +33,13 @@ def test_sets_auth_to_hhtp_basic_auth_method_with_api_key_as_username(self):
         self.assertEqual(self.chargebee_client.auth, requests.auth.HTTPBasicAuth('my_api_key', ''))
 
 
-class TestGetSubscriptions(TestBase):
+class TestGetPaginatedSubscriptions(TestBase):
     def setUp(self):
-        super(TestGetSubscriptions, self).setUp()
+        super(TestGetPaginatedSubscriptions, self).setUp()
```

Comment:
If we're only supporting Python 3 you can just do `super().setUp()`

Reply:
Oh I didn't know that. Nice :sunglasses: